tasks/install.yml

```
@@ -8,19 +8,7 @@
   delegate_to: 127.0.0.1
   run_once: true
 
-- name: Register if Cilium Helm repository is installed
-  command: helm search repo {{ cilium_chart_name }} -n {{ cilium_namespace }} --version ^{{ cilium_chart_version }}
-  register: cilium_repo_installed
-  changed_when: false
-  delegate_to: 127.0.0.1
-  run_once: true
-
-- name: Add cilium helm repository
-  command: helm repo add {{ cilium_chart_name }} {{ cilium_chart_url }}
-  changed_when: false
-  delegate_to: 127.0.0.1
-  run_once: true
-  when: "cilium_repo_installed.stdout.find(cilium_chart_name) == -1"
+- include_tasks: repo.yml
```

```suggestion
- include_tasks: helm_repository.yml
```
Please rename `repo.yml` to `helm_repository.yml`.